<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigationWindow</class>
 <widget class="QWidget" name="NavigationWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 247, 255);</string>
  </property>
  <widget class="QPushButton" name="recommendationButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 255);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Recommendation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="guidanceButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>240</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 255);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Guidence</string>
   </property>
  </widget>
  <widget class="QLabel" name="studentNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="milestoneButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 255);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Milestone</string>
   </property>
  </widget>
  <widget class="QLabel" name="gpaLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="semesterLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
